devices/somgoms.js

```diff
@@ -25,6 +25,15 @@ module.exports = [
         toZigbee: [tz.tuya_cover_control, tz.tuya_cover_options],
         exposes: [e.cover_position().setAccess('position', ea.STATE_SET)],
     },
+    {
+        zigbeeModel: ['sbordckq'],
+        model: 'TS0601_switch_cover',
```

Comment:
Is there a better model number written on the device itself or product page? (preferable something that gives results on google)

Reply:
I would have to remove it from the wall to make sure.
But googling revealed it's known by `SM-1CTW-EU`.
So I can name it that.

The similar looking (same line of) devices on the docs page follow the `TS0601_something` format.
[This one](https://www.zigbee2mqtt.io/devices/TS0601_switch_4_gang.html) for example.

What do you prefer?

Reply:
`SM-1CTW-EU` since that is the real model number.

Reply:
Done.
I also updated the docs pr.